mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for - Issue/147 user property panel (merge this second)

Closes #147 

To test:
- Run Mandatory Access Controls->MAC Integrity scenario
- Select users in scene or through User View
- Make sure User properties display correctly
